arangod/V8Server/v8-vocbase.cpp

```diff
@@ -769,8 +771,9 @@ static void JS_ExecuteAqlJson(v8::FunctionCallbackInfo<v8::Value> const& args) {
     queryId = TRI_NewServerSpecificTick();
   }
 
+  auto origin = transaction::OperationOriginREST{"executing query from JSON"};
```

Comment:
Why is this `OriginREST` rather than `OriginAql` as the other AQL calls in this file?

Reply:
Fixed